src/instrumentations/xhr.ts

```diff
@@ -753,7 +780,28 @@ export function instrumentXHR(config: XhrConfig) {
                 }
               }
             } else if (this.readyState === XMLHttpRequest.DONE) {
-              endSpan(EventNames.EVENT_READY_STATE_CHANGE, this);
+              if (config.networkBodyCapture && this.responseType === 'blob') {
+                try {
```

Comment:
curious what this try-catch is for

Reply:
probably leftover from experimenting, removed